sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/EventProcessorClient/EventProcessorClientTests.cs

```diff
@@ -666,24 +972,38 @@ private static async IAsyncEnumerable<PartitionEvent> MockPartitionEventEnumerab
         ///   Serves as a non-functional connection for testing consumer functionality.
         /// </summary>
         ///
-        private class MockConnection : EventHubConnection
+        internal class MockConnection : EventHubConnection
```

Comment:
Regarding the `internal` classes in this file, in the Event Hubs test project we usually do not import code directly from test files. Instead, we have an `Infrastructure` folder for things that are intended to be used in multiple places. However, if the code is really short, we don't bother and just repeat the code where it's needed.

For this reason, we are not expected to have `internal` classes or class members declarations in test files. I would suggest making the inner classes in this file `private` and, when necessary, just create a copy wherever it's required.

Reply:
will do!